modules/theora/video_stream_theora.cpp

```diff
@@ -245,6 +242,11 @@ void VideoStreamPlaybackTheora::set_file(const String &p_file) {
 
 	/* we're expecting more header packets. */
 	while ((theora_p && theora_p < 3) || (vorbis_p && vorbis_p < 3)) {
+#ifdef _MSC_VER
+		// Make exception for these assignments in conditional expression.
+#pragma warning(disable : 4706)
+#endif
```

Comment:
I went for silencing it here because decoding code is always hard to read anyway and I'm not sure how to refactor this to avoid the assignment without making it needlessly convoluted.